celery_worker.py

```diff
@@ -1,7 +1,7 @@
 import os
 from compair import create_app, celery
 
-app = create_app(skip_endpoints=True, skip_assets=True)
+app = create_app()
```

Comment:
oh, this is a mistake on my part. I originally had the email content generated in the delayed jobs (which required them to have access to the endpoints for url_for) so I removed it but celery had many problems using templates without a request context so I switch to just delivering the emails as a delayed task.